atlas/AtlasMl/atlasml/dependencies.py

```
@@ -0,0 +1,25 @@
+from fastapi import Depends
+from fastapi.requests import Request
+
+from atlasml.common.exceptions import (
+    PermissionDeniedException,
+    RequiresAuthenticationException,
+)
+from atlasml.config import APIKeyConfig, settings
```

Pylint (3.3.7)

[error] No name 'settings' in module 'atlasml.config'

(E0611)